libraries/lib-track/Track.cpp

```diff
@@ -219,13 +219,14 @@ void Track::DoSetLinkType(LinkType linkType, bool completeList)
       // Becoming unlinked
       assert(mpGroupData);
       if (HasLinkedTrack()) {
-         // Make independent copy of group data in the partner, which should
-         // have had none
-         auto partner = GetLinkedTrack();
-         assert(!partner->mpGroupData);
-         partner->mpGroupData =
-            std::make_unique<ChannelGroupData>(*mpGroupData);
-         partner->mpGroupData->mLinkType = LinkType::None;
+         if (auto partner = GetLinkedTrack()) {
```

Comment:
It is possible that `HasLinkedTrack` will succeed, but `GetLinkedTrack` still may return null pointer?

Reply:
Yes, that happens in this case.

A track was made as EmptyCopy of another, including the group data, and added to the end of the TrackList.  It was stereo group data but the right channel was not yet also added.  So in fact link consistency check criteria are temporarily violated.  But then the inconsistency is repaired by resetting the link type, but keeping any other group data.